src/features/user/MFV2/MyContributions/ProjectItemCard.tsx

```diff
@@ -93,8 +93,8 @@ const ProjectItemCard = ({
                 projectPurpose={project.purpose}
                 buttonText={
                   pageType === 'public'
-                    ? tProfile('donate')
-                    : tProfile('donateAgain')
+                    ? tProfile('donateAgain')
+                    : tProfile('donate')
```

Comment:
Reverse this - the Donate Again button can only be seen when the profile type is private